packages/@aws-cdk/aws-autoscaling/test/integ.warm-pool.ts

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import * as autoscaling from '../lib';
+
```

Comment:
can you add reproduction steps to show that the integ test not only deploys but can be interfaced via api? take a look at some of our other integ tests, they _should_ have reproduction steps.

Reply:
Will do.

Reply:
Actually, can you elaborate what " reproduction step" mean here? Can you share an example?

Reply:
Example: https://github.com/aws/aws-cdk/blob/8dd93a8936ddc74b799e0c6dafbe0511ee687e71/packages/@aws-cdk/aws-synthetics/test/integ.canary.ts#L14